src/esockd_listener.erl

```diff
@@ -117,6 +125,9 @@ handle_cast(Msg, State) ->
     error_logger:error_msg("[~s] Unexpected cast: ~p", [?MODULE, Msg]),
     {noreply, State}.
 
+handle_info({'EXIT', LSock, _}, #state{lsock = LSock} = State) ->
+    error_logger:error_msg("~s Lsocket ~p closed", [?MODULE, LSock]),
```

Comment:
```suggestion
    error_logger:error_msg("~s listener socket ~p closed", [?MODULE, LSock]),
```

Reply:
We have `?MODULE` in the log, so we know it is the listener.

 The listener has no other socket I think.